UnitsNet/UnitDefinitions/RotationalStiffness.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "RotationalStiffness",
+  "BaseUnit": "NewtonMeterPerRadian",
+  "XmlDoc": "The resistance offered by an elastic body to deformation by means of rotation.",
+  "Units": [
+    {
+      "SingularName": "NewtonMeterPerRadian",
+      "PluralName": "NewtonMetersPerRadian",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Kilo", "Mega" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "Nm/rad" ]
```

Comment:
We  have a [guideline](https://github.com/angularsen/UnitsNet/blob/master/CONTRIBUTING.md#unit-abbreviations) to use this notation `N·m`. It's also how [`Torque`](https://github.com/angularsen/UnitsNet/blob/master/UnitsNet/UnitDefinitions/Torque.json#L41) has defined it.

You can keep both variations if you want to be able to `Parse()` both, but the first one is used by `ToString()`:
```
"Abbreviations": [ "N·m/rad", "Nm/rad" ]
```